spring-integration-http/src/main/resources/org/springframework/integration/http/config/spring-integration-http-5.0.xsd

```diff
@@ -726,6 +859,60 @@
 		</xsd:attribute>
 	</xsd:complexType>
 
+	<xsd:attributeGroup name="asyncHttpOutboundCommonAttributes">
+		<xsd:attribute name="async-rest-template" type="xsd:string">
```

Comment:
OK. Let it be!
Since you have done already this hard work for the XML support, I think we could live with the solution.
More over I see that even `AsyncRestTemplate` uses `Async` naming for `setAsyncRequestFactory(AsyncClientHttpRequestFactory asyncRequestFactory)`.
So, to be consistent, let's go the `async` way! 😄